cmd/soroban-rpc/internal/config/options.go

```diff
@@ -211,18 +211,22 @@ func (cfg *Config) options() ConfigOptions {
 		},
 		{
 			Name: "event-retention-window",
-			Usage: fmt.Sprintf("configures the event retention window expressed in number of ledgers,"+
-				" the default value is %d which corresponds to about 24 hours of history", ledgerbucketwindow.DefaultEventLedgerRetentionWindow),
+			Usage: fmt.Sprintf(
+				"configures the event retention window expressed in number of ledgers,"+
+					" the default value is %d which corresponds to about 24 hours of history",
+				ledgerbucketwindow.DefaultEventLedgerRetentionWindow),
 			ConfigKey:    &cfg.EventLedgerRetentionWindow,
 			DefaultValue: uint32(ledgerbucketwindow.DefaultEventLedgerRetentionWindow),
 			Validate:     positive,
 		},
 		{
 			Name: "transaction-retention-window",
-			Usage: "configures the transaction retention window expressed in number of ledgers," +
-				" the default value is 1440 which corresponds to about 2 hours of history",
+			Usage: fmt.Sprintf(
+				"configures the transaction retention window expressed in number of ledgers,"+
+					" the default value is %d which corresponds to about 24 hours of history",
+				ledgerbucketwindow.OneDayOfLedgers),
 			ConfigKey:    &cfg.TransactionLedgerRetentionWindow,
-			DefaultValue: uint32(1440),
+			DefaultValue: uint32(ledgerbucketwindow.OneDayOfLedgers),
```

Comment:
oh we should undo this?

Reply:
Yeah I think so, users can set it higher if they want to. Do we want to add some upper bound here? like dont go beyond 7 days or something? I know we have not added those restrictions for in-memory stores but might be a good idea.

Reply:
nah I think it's fine, if you have disk why not :shrug: